### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,12 +41,12 @@
     "clean-css": "2.0.x",
     "commander": "2.0.x",
     "enhance-css": "0.6.x",
-    "less": "1.4.x",
+    "less": "1.5.x",
```
Comment (GoalSmashers) [inline package.json:44]: If you update something in `dependencies` please add it to `History.md` too.

**Comment by GoalSmashers** [comment]
Looks good beside that.

**Comment by XhmikosR** [comment]
I always leave this stuff to you since it's your projects :)

**Comment by GoalSmashers** [comment]
Well if you don't want to then fine.

**Pull request merged**